src/node/db/SecurityManager.js

```diff
@@ -96,7 +96,7 @@ exports.checkAccess = async function(padID, sessionCookie, token, password)
   // get information about all sessions contained in this cookie
   if (sessionCookie) {
     let groupID = padID.split("$")[0];
-    let sessionIDs = sessionCookie.split(',');
+    let sessionIDs = sessionCookie.replace(/"/g, '').split(',');
```

Comment:
If this is meant to always remove the double quotes (`"`), we could achieve it without firing a regexp:

```js
replace('"', '')
```

Reply:
This removes only first quote. We need to remove both quotes, at the beginning and at the end.

Reply:
Yup